acceptance_tests/features/sign_in_internal.feature

```diff
@@ -9,34 +9,43 @@ Feature: Internal user signs in
   Scenario: User signs in correctly
     Given the user has an active account and is assigned a username and password
     When they enter the correct username and password
+    And they click the sign in button
     Then the user is directed to their home page
 
   Scenario: User attempts sign in with incorrect username and receives authentication error
     Given the user has an active account and is assigned a username and password
     When they enter an incorrect username and correct password
+    And they click the sign in button
     Then the user is notified that an authentication error has occurred
 
   Scenario: User attempts sign in with incorrect password and receives authentication error
     Given the user has an active account and is assigned a username and password
     When they enter a correct username and incorrect password
+    And they click the sign in button
     Then the user is notified that an authentication error has occurred
 
   Scenario: User attempts sign in and receives authentication error
     Given the user has an active account and is assigned a username and password
     When they enter an incorrect username and password
+    And they click the sign in button
     Then the user is notified that an authentication error has occurred
 
+  @skip
   Scenario: User attempts sign in and is notified that they are required to enter a password
     Given the user has an active account and is assigned a username and password
     When they enter a correct username and no password
+    And they click the sign in button
     Then the user is notified that a password is required
 
+  @skip
```

Comment:
Is this scenario meant to be skipped?

Reply:
Yes , I have raised another ticket for these as it takes a bigger change.